examples/simulator-1/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM reachsh/runner:0.1
+
+ENV NODE_VERSION=16.13.0
```

Comment:
This should use the one in `/DEPS` by inclusion, but actually `runner` already has `node`, so you should be able to use it